src/elements/public/Admin/Log/ErrorLog.ts

```diff
@@ -0,0 +1,95 @@
+import type * as FoxySDK from '@foxy.io/sdk';
+import { ScopedElementsMap, ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { NucleonElement } from '../../NucleonElement';
+import { html } from 'lit-html';
+import { CSSResult, CSSResultArray, PropertyDeclarations, TemplateResult } from 'lit-element';
+import { ErrorEntry } from './ErrorEntry';
+import { ButtonElement } from '@vaadin/vaadin-button';
+import { Themeable } from '../../../../mixins/themeable';
+
+type ErrorEntries = FoxySDK.Backend.Rels.ErrorEntries;
+type Data = FoxySDK.Core.Resource<ErrorEntries>;
+
+export class ErrorLog extends ScopedElementsMixin(NucleonElement)<Data> {
+
+  static get properties(): PropertyDeclarations {
+    return {
+      ...super.properties,
+      showHidden: {
+        type: Boolean
+      },
+    };
+  }
+
+  public static get scopedElements(): ScopedElementsMap {
+    return {
+      'foxy-error-entry': ErrorEntry,
+      'foxy-spinner': customElements.get('foxy-spinner'),
+      'iron-icon': customElements.get('iron-icon'),
+      'vaadin-button': ButtonElement,
+    }
+  }
+
+  static get styles(): CSSResult | CSSResultArray {
+    return [
+      Themeable.styles,
+    ];
+  }
+
+  showHidden = false;
+
+  render(): TemplateResult {
```

Comment:
**suggestion:** slightly different design for error log

How about changing the “Show archived” functionality to tabs? See the attached design. In that case the top-level element would be `ErrorEntries` (`foxy-error-entries`) instead of `ErrorLog` (`foxy-error-log`) to follow the naming conventions, and it’d use `Tabs` and `CollectionPages`.

<img width="1440" alt="List" src="https://user-images.githubusercontent.com/47783757/119495028-9fcb0e80-bd6a-11eb-99e2-4663b61a53bf.png">

Reply:
@pheekus I like that approach. @ndvo looks more clear than trying to finagle less-clear UI.